### DISCUSSION TITLE
Add part# to pc-part-dataset

### DISCUSSION THREAD

**Discussion by bacleclement** [docyx/pc-part-dataset#8] [Q&A]
Hi, I fork this project and I tried to add part # to pc-part-dataset but when I used Puppeteer to scrape it it was impossible to find this balise. Do u have an idea? Thanks a lot. Bye!
<img width="1403" alt="Capture d’écran 2024-03-06 à 21 07 34" src="https://github.com/docyx/pc-part-dataset/assets/32833885/578ed59f-0b27-49f0-bd38-0d8cc88e209a">

**Comment by docyx** [comment]
It can be done but the scraper would have to be modified to fetch the spec sheet for each part, which translates to about ~60,000 requests (assuming you want everything). I can definitely do this, and since this information is static I can also try to implement a smarter updater that only fetches what has been added since the last scrape. I'll try to work on this this weekend.

**Reply by bacleclement** [reply]
Hi, yes It will be very helpful! Thanks a lot :) This project is very helpful to me.
